sourcemash/api/categories.py

```diff
@@ -44,24 +79,13 @@ def get(self):
                              .all()
 
         for category, count in categories:
-            read_item_count = UserItem.query \
-                                .filter(UserItem.user_id==current_user.id) \
-                                .join(UserItem.item) \
-                                .filter(Item.cats.contains(category)) \
-                                .filter(~UserItem.unread) \
-                                .count()
-
-            category.item_count = count
-            category.unread_count = category.item_count - read_item_count
 
-            unsubscribed_item = Item.query.filter(Item.cats.contains(category)) \
+            if count > MIN_ITEM_COUNT:
+                unsubscribed_item = Item.query.filter(Item.cats.contains(category)) \
```

Comment:
I was thinking about it, but I still like the feature. I think ideally it should be a setting that users turn on & off (just a column on the user table), but it's worth keeping.